webpack/assets/javascripts/react_app/components/ModelsTable/index.js

```diff
@@ -0,0 +1,35 @@
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import ModelsTable from './ModelsTable';
+import { schema } from './ModelsTableSchema';
+import reducer from './ModelsTableReducer';
+import * as actions from './ModelsTableActions';
+
+const mapStateToProps = ({ models }) => ({ ...models });
+
+const mapDispatchToProps = dispatch => bindActionCreators(actions, dispatch);
+
+const mergeProps = (stateProps, dispatchProps, { data }) => {
+  const { getModels } = dispatchProps;
+  const { sortBy, sortOrder } = stateProps;
+  const { url } = data;
+  return Object.assign({}, data, stateProps, dispatchProps, {
+    columns: schema({
+      getModels,
+      url,
+      sortBy,
+      sortOrder,
+    }),
+    filterFnc: (store = {}, displayModels = []) =>
```

Comment:
afaik on each react's `render()` this function's ref will be changed, which might lead to poor performance. 
is there a reason to have this `filterFnc` here?